internal/utils/misc.go

```diff
@@ -157,6 +158,43 @@ var (
 	ErrNotRunning  = errors.Errorf("%s is not running.", Aqua("supabase start"))
 )
 
+func GetSeedsFilepaths() string {
+	if seedPaths, _ := GetSeedFiles(afero.NewOsFs()); seedPaths != nil {
+		return fmt.Sprintf("%v", seedPaths)
+	}
+	return DefaultSeedDataPath
+}
+
+/*
+** Match the glob patterns from the config to get
+** a deduplicated array of all migrations files to apply
+** in the declared order
+ */
+func GetSeedFiles(fsys afero.Fs) ([]string, error) {
+	seedPaths := Config.Db.Seed.Path
+	fileSet := make(map[string]struct{})
+	var files []string
+	for _, pattern := range seedPaths {
+		fullPattern := filepath.Join(SupabaseDirPath, pattern)
+		matches, err := afero.Glob(fsys, fullPattern)
+		if err != nil {
+			return nil, errors.Errorf("failed to apply glob pattern for %w", err)
+		}
```

Comment:
How about throwing an error when `len(matches) == 0`? This helps users catch typos in their seed paths.

Reply:
Yeah tough about that as well.
I was thinking about logging a warning in that case something like `Warning: Your pattern XXXX matched 0 seeds files.` but I can see the benefit of an early fail as well.

Reply:
true, happy to go with a warning too.